.rubocop.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 AllCops:
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 3.0
```

Comment:
why would this change for a test?

Reply:
That's the version of Ruby we run on the beanstalks, and we run Rubocop along with our tests.